include/zephyr/drivers/cellular.h

```diff
@@ -96,8 +124,21 @@ enum cellular_modem_info_type {
  * @retval -ENOTSUP if API is not supported by cellular network device.
  * @retval Negative errno-code otherwise.
  */
-int cellular_configure_networks(const struct device *dev, const struct cellular_network *networks,
-				uint8_t size);
+__syscall int cellular_configure_networks(const struct device *dev,
```

Comment:
Does this API work properly when called from userspace? I have #65746 where I am trying to remove the syscall calls from phy interface as userspace cannot access the device.
So is this really tested to work from userspace application or did you added the syscall because many of the other APIs have it?

Reply:
I added it with the intention of adding handlers later. I have removed it for now :)